backoffice/templates/backoffice/components/driver_profile_card.html

```diff
@@ -0,0 +1,63 @@
+<div class="rounded-lg shadow w-96 px-3 py-3">
+    <h2 class="text-xl font-bold text-gray-800 mb-4">{{ profile.full_name }}</h2>
+    
+    <p class="text-gray-600 mb-2">
+        <strong>Created At:</strong> {{ profile.created_at }}
+    </p>
+    <p class="text-gray-600 mb-2">
+        <strong>Updated At:</strong> {{ profile.updated_at }}
+    </p>
+    <p class="text-gray-600 mb-2">
+        <strong>Submitted At:</strong> {{ profile.submitted_at }}
+    </p>
+    <p class="text-gray-600 mb-2">
+        <strong>Approved At:</strong> {{ profile.approved_at }}
+    </p>
+    <p class="text-gray-600 mb-2">
+        <strong>Expires At:</strong> {{ profile.expires_at }}
+    </p>
+
+    <div class="mt-4">
+        <h3 class="font-semibold text-gray-700 mb-2">Driver Details</h3>
+        <p><strong>Licence Number:</strong> {{ profile.licence_number }}</p>
+        <p><strong>Issue Date:</strong> {{ profile.licence_issue_date }}</p>
+        <p><strong>Expiry Date:</strong> {{ profile.licence_expiry_date }}</p>
+    </div>
+
+    <div class="mt-4">
+        <h3 class="font-semibold text-gray-700 mb-2">Address</h3>
+        <p>{{ profile.address_line_1 }}</p>
+        {% if profile.address_line_2 %}<p>{{ profile.address_line_2 }}</p>{% endif %}
+        {% if profile.address_line_3 %}<p>{{ profile.address_line_3 }}</p>{% endif %}
+        <p>{{ profile.postcode }}</p>
+    </div>
+
+    <div class="mt-4 grid grid-cols-1 sm:grid-cols-2 gap-4">
+        <div>
+            <h3 class="font-semibold text-gray-700 mb-2">Licence Images</h3>
+            <p><strong>Front:</strong> <a href="/media/{{ profile.licence_front }}" target="_blank" class="text-blue-500">View</a></p>
```

Comment:
Are these links correct? I have a suspicion they are not but it's been a long time so I may be mistaken. They should be constructed the same way as the driver profile approval forms, and should not be a link if there's no value in the field but should instead say "not provided" or similar.